azure-pipelines/build.yml

```diff
@@ -79,7 +79,7 @@ steps:
   inputs:
     command: test
     projects: test/Microsoft.Kusto.ServiceLayer.UnitTests
-    arguments: '--logger "trx;LogFileName=xunit.trx"'
+    arguments: '--logger "trx;LogFileName=nunit.trx" --collect "Code coverage"'
```

Comment:
What was the issue here? Is this something that should also be done for SqlTools.ServiceLayer.UnitTests?

Reply:
It could be done for the SqlTools Service. This generates the code coverage for unit tests for the project. I changed the LogFileName so it wasn't the same as SqlTools and also because Kusto uses nunit not xunit.

Reply:
Would suggest naming it `Kusto.ServiceLayer.UnitTests.trx` or something then to make it that much clearer.

Would you mind adding this to the Sql Tools service layer as well (and updating the name similarly there as well)? Seems like a good thing to have.

Reply:
Here's a link to this build. It now contains a "Code Coverage" tab and has a Coverage stat with the test pass stat.

![image](https://user-images.githubusercontent.com/63619224/135510795-65cb4634-a3a1-4a82-aefd-3b18b98142dc.png)


https://mssqltools.visualstudio.com/CrossPlatBuildScripts/_build/results?buildId=121958&view=results